stdlib/public/core/SwiftNativeNSArray.swift

```diff
@@ -129,6 +141,188 @@ extension __SwiftNativeNSArrayWithContiguousStorage: _NSArrayCore {
   internal func copy(with _: _SwiftNSZone?) -> AnyObject {
     return self
   }
+  
+  @objc(mutableCopyWithZone:)
+  internal func mutableCopy(with _: _SwiftNSZone?) -> AnyObject {
+    let arr = [AnyObject]._bridgeFromObjectiveCAdoptingNativeStorageOf(self)
```

Comment:
Not 100% certain this is the best way to get an [AnyObject] from inside its storage

Reply:
We could just call `Array<AnyObject>(_ContiguousArrayBuffer(self))`, I think -- except maybe we'd need to move this down the class hierarchy a bit.